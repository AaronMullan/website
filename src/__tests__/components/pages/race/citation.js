import React from 'react'
import renderer from 'react-test-renderer'
import Citation from '~components/pages/race/citation'

describe('Components : Race : Citation', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Citation>
          <p>component children</p>
        </Citation>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
