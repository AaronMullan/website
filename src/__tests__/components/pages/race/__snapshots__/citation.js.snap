// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Race : Citation renders correctly 1`] = `
<cite>
  <p>
    component children
  </p>
</cite>
`;
